app/helpers/container_helper/textual_summary.rb

```diff
@@ -4,12 +4,12 @@ module ContainerHelper::TextualSummary
   #
 
   def textual_group_properties
-    items = %w(image name state restart_count backing_ref image_ref)
+    items = %w(name state restart_count backing_ref)
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
   def textual_group_relationships
-    items = %w(ems container_replicator container_group container_project)
+    items = %w(ems container_replicator container_group container_project container_image)
```

Comment:
>> The idea is to go from the most generic to the most specific. So basically in an ems you have multiple container_projects, each project may have container_replicators that are pointing to container_groups that are using container_images.
> 
> pods are not using images, containers are.

Containers are not in this list (that's why I haven't mentioned them). As the there are no `routes`, if there were I would have mentioned them as well (`projects` contain `routes`, `routes` point to `services`, `services` to `replicators` ...)

Order was suggested by John and Mike several months ago.